test/unit/lib/helpers/regex.test.js

```diff
@@ -0,0 +1,59 @@
+const globals = require(`${__dirname}/../../../../app/globals`) // Always required
+const regex = require(`${__dirname}/../../../../app/lib/helpers/regex`)
+
+describe('Regex: URL', () => {
+  it('should export object', () => {
+    expect(regex.url).toBeInstanceOf(Object)
+  })
+
+  describe('URL test http', () => {
+    const http = 'http://example.com'
+    const match = regex.url.exec(http)
+    it (`Should not return null for '${http}'`, () => {
+      expect(match).not.toBeNull()
+    })
+    it (`Should have a first argument of '${http}'`, () => {
+      expect(match[0]).toBe('http://example.com')
+    })
+    it (`Should have a first argument of 'http'`, () => {
+      expect(match[1]).toBe('http')
+    })
+  })
+
+  describe('URL test https', () => {
+    const https = 'https://example.com'
+    const match = regex.url.exec(https)
+    it (`Should not return null for '${https}'`, () => {
```

Comment:
Looks good! The only thing I'd change is add a line break between each test, so that it's consistent with our other test suites. And also remote the space on the `it()` call.